src/EditorFeatures/Test2/Rename/RenameEngineTests.VisualBasicConflicts.vb

```diff
@@ -12,7 +12,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
                 _outputHelper = outputHelper
             End Sub
 
-            <WpfFact(Skip:="798375, 799977")>
+            <WpfFact>
```

Comment:
It seems odd that this test has a assertion the WriteLine invocation has a conflict. @dpoeschl, thoughts on this?